userspace/libscap/engine/gvisor/gvisor.cpp

```diff
@@ -76,9 +77,9 @@ static int32_t gvisor_next(struct scap_engine_handle engine, scap_evt **pevent,
 	return engine.m_handle->next(pevent, pcpuid);
 }
 
-static bool gvisor_match(scap_open_args* open_args)
+static bool gvisor_match(scap_open_args* oargs)
```

Comment:
Do we really need such a match API when we can just `strncmp` ?
Perhaps it is better to change the `match` API to just `engine_name` and let scap call
`strcmp(oargs->engine_name, handle->engine_name()) == 0`
?